src/components/ApiService/ApiService.js

```diff
@@ -0,0 +1,18 @@
+import axios from 'axios'
```

Comment:
ApiService is a service, not a component so it would be better if it is in a separate directory.